### PR TITLE
Feat/model rapip

### PR DESCRIPTION
**Pull request by Rafif1103**
please review the relations

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -11,12 +11,174 @@ datasource db {
...
+  pertandingans Tim_Pertandigan[]
+  undangans     Undangan[]
+  pendaftarans  Pendaftaran[]
+}
+
+model Pemain {
+  username String @id
+  user     User   @relation(fields: [username], references: [username])
+
+  nama_tim String? @unique
```
Comment (mhadziqrazin) [inline prisma/schema.prisma]: please revise. no need unique constraint

Comment (triasaf) [reply prisma/schema.prisma]: i agree

Comment (Rafif1103) [reply prisma/schema.prisma]: thanks for the input

**Review by triasaf — APPROVED** [review]
all good 👍. Thanks Rafif!

**Review by mhadziqrazin — APPROVED** [review]

**Pull request merged**